When output format is empty string, kubectl panic: runtime error: invalid memory address or nil pointer dereference

```shell
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"2", GitVersion:"v1.2.2", GitCommit:"528f879e7d3790ea4287687ef0ab3f2a01cc2718", GitTreeState:"clean"}
Server Version: version.Info{Major:"1", Minor:"2", GitVersion:"v1.2.2", GitCommit:"528f879e7d3790ea4287687ef0ab3f2a01cc2718", GitTreeState:"clean"}

$ kubectl config view --output=""
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x28 pc=0x2c1b36]

goroutine 1 [running]:
k8s.io/kubernetes/pkg/kubectl.(*VersionedPrinter).PrintObj(0xc2080fcc80, 0x1c93db0, 0xc208192fc0, 0x1c8fff0, 0xc20804e008, 0x0, 0x0)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/kubectl/resource_printer.go:189 +0x426
k8s.io/kubernetes/pkg/kubectl/cmd/config.ViewOptions.Run(0x1c949d0, 0xc208050eb0, 0x100, 0x1c8fff0, 0xc20804e008, 0x1c94a88, 0xc2080fcc80, 0x0, 0x0)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/kubectl/cmd/config/view.go:113 +0x19d
k8s.io/kubernetes/pkg/kubectl/cmd/config.func·009(0xc2081a6a00, 0xc208135490, 0x0, 0x1)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/kubectl/cmd/config/view.go:77 +0x3e2
github.com/spf13/cobra.(*Command).execute(0xc2081a6a00, 0xc208135400, 0x1, 0x1, 0x0, 0x0)
	/go/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/spf13/cobra/command.go:572 +0x82f
github.com/spf13/cobra.(*Command).ExecuteC(0xc208073600, 0xc2081a6a00, 0x0, 0x0)
	/go/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/spf13/cobra/command.go:662 +0x4db
github.com/spf13/cobra.(*Command).Execute(0xc208073600, 0x0, 0x0)
	/go/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/spf13/cobra/command.go:618 +0x3a
k8s.io/kubernetes/cmd/kubectl/app.Run(0x0, 0x0)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubectl/app/kubectl.go:28 +0x101
main.main()
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubectl/kubectl.go:28 +0x2f

goroutine 5 [chan receive]:
github.com/golang/glog.(*loggingT).flushDaemon(0x1a3f640)
	/go/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/golang/glog/glog.go:879 +0x78
created by github.com/golang/glog.init·1
	/go/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/golang/glog/glog.go:410 +0x2a7

goroutine 13 [syscall]:
os/signal.loop()
	/usr/src/go/src/os/signal/signal_unix.go:21 +0x1f
created by os/signal.init·1
	/usr/src/go/src/os/signal/signal_unix.go:27 +0x35

$ kubectl config view -o ""
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x28 pc=0x2c1b36]

goroutine 1 [running]:
k8s.io/kubernetes/pkg/kubectl.(*VersionedPrinter).PrintObj(0xc2080c8c40, 0x1c93dd8, 0xc20819cfc0, 0x1c8fff0, 0xc208048008, 0x0, 0x0)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/kubectl/resource_printer.go:189 +0x426
k8s.io/kubernetes/pkg/kubectl/cmd/config.ViewOptions.Run(0x1c94ab8, 0xc20800af50, 0x100, 0x1c8fff0, 0xc208048008, 0x1c94b70, 0xc2080c8c40, 0x0, 0x0)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/kubectl/cmd/config/view.go:113 +0x19d
k8s.io/kubernetes/pkg/kubectl/cmd/config.func·009(0xc2081aea00, 0xc2081a5300, 0x0, 0x2)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/kubectl/cmd/config/view.go:77 +0x3e2
github.com/spf13/cobra.(*Command).execute(0xc2081aea00, 0xc2081a5140, 0x2, 0x2, 0x0, 0x0)
	/go/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/spf13/cobra/command.go:572 +0x82f
github.com/spf13/cobra.(*Command).ExecuteC(0xc20807d600, 0xc2081aea00, 0x0, 0x0)
	/go/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/spf13/cobra/command.go:662 +0x4db
github.com/spf13/cobra.(*Command).Execute(0xc20807d600, 0x0, 0x0)
	/go/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/spf13/cobra/command.go:618 +0x3a
k8s.io/kubernetes/cmd/kubectl/app.Run(0x0, 0x0)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubectl/app/kubectl.go:28 +0x101
main.main()
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubectl/kubectl.go:28 +0x2f

goroutine 5 [chan receive]:
github.com/golang/glog.(*loggingT).flushDaemon(0x1a3f640)
	/go/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/golang/glog/glog.go:879 +0x78
created by github.com/golang/glog.init·1
	/go/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/golang/glog/glog.go:410 +0x2a7

goroutine 13 [syscall]:
os/signal.loop()
	/usr/src/go/src/os/signal/signal_unix.go:21 +0x1f
created by os/signal.init·1
	/usr/src/go/src/os/signal/signal_unix.go:27 +0x35
```

--- a/pkg/kubectl/cmd/config/view.go
+++ b/pkg/kubectl/cmd/config/view.go
@@ -67,6 +67,10 @@ func NewCmdConfigView(out io.Writer, ConfigAccess ConfigAccess) *cobra.Command {
 				fmt.Printf("--output wide is not available in kubectl config view; reset to default output format (%s)\n\n", defaultOutputFormat)
 				cmd.Flags().Set("output", defaultOutputFormat)
 			}
+			if outputFormat == "" {
+				fmt.Printf("reset to default output format (%s) as --output is empty", defaultOutputFormat)
+				cmd.Flags().Set("output", defaultOutputFormat)
+			}
 
 			printer, _, err := cmdutil.PrinterForCommand(cmd)
 			cmdutil.CheckErr(err)

